chemicaltoolbox/chembl/chembl.xml

```diff
@@ -0,0 +1,118 @@
+<tool id="chembl" name="Search ChEMBL database" version="0.1.0">
+    <description>for compounds which are similar to a SMILES string</description>
+    <requirements>
+        <requirement type="package" version="0.9.31">chembl_webresource_client</requirement>
+    </requirements>
+    <command detect_errors="exit_code"><![CDATA[
+        python -W ignore '$__tool_directory__/chembl.py'
```

Comment:
What does this `-W ignore` filter?

Reply:
The python package raises a warning which I'd like to hide from the user. That flag suppresses it.